assets/src/apps/delivery/layouts/deck/DeckLayoutFooter.tsx

```diff
@@ -582,6 +583,7 @@ const DeckLayoutFooter: React.FC = () => {
             <HistoryNavigation />
           </>
         )}
+        {isLegacyTheme && <HistoryNavigation />}
```

Comment:
does this still need a condition? it seems like it's here in both cases

Reply:
good catch... fixed.